index.html

```diff
@@ -0,0 +1,322 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio-Jorum Odiemo</title>
+    <link rel="stylesheet" href="styles.css">
+</head> 
+<body>
+    <header class="Top">
+        <a href=""><img src="images/MyLogo.png" alt="My Logo" ></a>
+        <nav class="navbar">            
+            <ul class="navbarmenuul">
+                <li class="navbarmenuli">
+                  <a href="#works" class="navbarmenulink">Portfolio</a>
+                </li>
+                <li class="navbarmenuli">
+                  <a href="#about" class="navbarmenulink">About</a>
+                </li>
+                <li class="navbarmenuli">
+                  <a href="#contact" class="navbarmenulink">Contact</a>
+                </li>
+              </ul>
```

Comment:
- You have done a good job so far, But please there are some changes not implemented, as requested previously, the mobile view of the webpage is not looking like that of the Figma design  

#### Mobile view
- [Optional] Please make sure the background image on the header section does not overlap the text.
- Please align the elements on the `project section` properly. 
  - PLease the button needs to be aligned properly
  - Please  use the proper `font-size` and `font-family` as used in the Figma design 
  - PLease resize the projects card to fit the mobile screen
- Please align the `About section` properly
  - Please use the proper font size and font family in the about section
  - Please use the proper design for the button `Get my resume`
  - Please align the languages section properly as in the Figma design.
  
- Please on the `contact section` proper position the background image so it does not go out of the main contact section container 

##### Figma design 
https://user-images.githubusercontent.com/58771507/173202634-17b4d0e5-4075-4d12-aa3e-aa5dc3ed0184.mp4

##### Your design 
https://user-images.githubusercontent.com/58771507/173202757-1e91fdb1-6e32-4fb5-9695-f288a0a329f2.mp4